### PR TITLE
Env support for single module

### PR DESCRIPTION
**Pull request by SLdragon**

### CODE REVIEW

File: iotedgehubdev/edgemanager.py [hunk 1]
```diff
@@ -434,6 +434,8 @@ def _start_edge_hub(self, edgedockerclient, edgeHubConnStr, routes, mount_base):
             EdgeManager.HUB_SSLCRT_ENV,
             'IotHubConnectionString={0}'.format(edgeHubConnStr)]
         hubEnv.extend(routes)
+        hubEnv.extend(list(envs))
```
Comment (adashen) [inline iotedgehubdev/edgemanager.py]: we need check the env input format

Comment (SLdragon) [reply iotedgehubdev/edgemanager.py]: updated

File: iotedgehubdev/cli.py [hunk 2]
```diff
@@ -239,7 +248,11 @@ def start(inputs, port, deployment, verbose, host):
             else:
                 input_list = [input_.strip() for input_ in inputs.strip().split(',')]
 
-            edge_manager.start_singlemodule(input_list, port)
+            for env in environment:
```
Comment (adashen) [inline iotedgehubdev/cli.py]: if user didn't specify -e, will it be null for environment? if so, there will be exception

File: iotedgehubdev/cli.py [hunk 3]
```diff
@@ -239,7 +248,11 @@ def start(inputs, port, deployment, verbose, host):
...
+                if re.match(r'^[a-zA-Z][a-zA-Z0-9_]*?=.*$', env) is None:
```
Comment (adashen) [inline iotedgehubdev/cli.py]: so "a=" is valid and no exception?

Comment (SLdragon) [reply iotedgehubdev/cli.py]: yes ,it is valid

File: iotedgehubdev/cli.py (hunk 2 above)
Comment (SLdragon) [reply iotedgehubdev/cli.py]: it was handled by the click module, and would pass environment as empty tuple, so it would not throw exception

**Pull request merged**